src/index.js

```diff
@@ -19,35 +19,35 @@ export default function loader(content) {
   });
 
   let outputPath = '';
+  const filePath = this.resourcePath;
```

Comment:
`const filePath` => `const resourcePath`

Reply:
Isn't `url` (L15) already returning the `filePath`?  What is missing there ?

Reply:
The `url` is relative path, the filePath is a full absolute path. I need to return outputPath depends on the path info in the full path.

Reply:
For this one: `const filePath => const resourcePath`, I just move this line from below